reactive/etcd.py

```diff
@@ -177,6 +177,17 @@ def follower_config_changed():
     close_open_ports()
 
 
+@when('snap.installed.etcd')
+@when('config.changed.service_network_interface')
+@when_not('etcd.installed')
```

Comment:
The `etcd.installed` state is actually an old carryover from pre-snap deployments. Today's deployments do not have it set.

A lot of handlers have `@when_not('etcd.installed')` slapped on them as a way to prevent any handlers from running until the user has migrated to snaps.

There's a note about it here:

https://github.com/juju-solutions/layer-etcd/blob/7307aa988db33687aa8707ae646d833f6bb80886/reactive/etcd.py#L45-L49

Reply:
now on revision 126 ><